matsim/scenariogen/data/__init__.py

```diff
@@ -327,16 +332,24 @@ class Trip:
     purpose: Purpose
     sd_group: SourceDestinationGroup
     valid: bool
+    dep_district: str = ""
+    arr_district: str = ""
+    arrival: int = 0
 
 
 @dataclass
 class Activity:
     """ Activity information (including leg) """
+    # all leg information relates to the leg leading to the activity
```

Comment:
You may add this in the docstring and not only as comment.

Reply:
can you explain the difference?

Reply:
The docstring is like javadoc and will probably be shown by the idea if one looks up this class.